inference-engine/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -135,19 +135,19 @@ static void fill_data_roi(float *data, size_t size, const uint32_t range, const
         if (data[i + 3] < data[i + 1]) {
             std::swap(data[i + 1], data[i + 3]);
         }
-        if (data[i + 1] < 0)
+        if (data[i + 1] < 0 || data[i + 1] > width - 1)
             data[i + 1] = 0;
```

Comment:
if `data[i + 1] > width - 1`, shouldn't it be `data[i + 1] = static_cast<float>(width - 1);` ?

Reply:
yes, I think you are right, I should fix this place.